x/concentrated-liquidity/lp.go

```diff
@@ -130,7 +140,7 @@ func (k Keeper) withdrawPosition(ctx sdk.Context, owner sdk.AccAddress, position
 	}
 
 	// If the position is still frozen, claim and forfeit any accrued incentives for the position.
-	isPositionFrozen := position.JoinTime.Add(position.FreezeDuration).After(ctx.BlockTime())
+	isPositionFrozen := k.IsPositionStillFrozen(ctx, position.JoinTime, position.FreezeDuration)
```

Comment:
Now that we're removing freezeDuration, we should be able to remove this check and just call claimAllIncentivesForPosition. It might make sense to rebase this PR to the latest CL PR that has these changes.